### PR TITLE
OSX Notification Center

### PR DESCRIPTION
**Pull request by heilerich**
Added a simple bridge that creates native OSX notifications using PyObjC (available from PyPi). With this you can receive gntp notifications on your Mac without installing Growl.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -35,6 +35,7 @@
             'local = regrowl.bridge.local:LocalNotifier',
             'subscribe = regrowl.bridge.subscribe:SubscribelNotifier',
             'udp = regrowl.bridge.udp:UDPNotifier',
+            'netcent = regrowl.bridge.netcent:NetCenter'
```
Comment (kfdm) [inline setup.py]: For OCD reasons, could you sort this alphabetically and end the line in a "," ? :)

File: regrowl/bridge/netcent.py [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+class NetCenter(ReGrowler):
+    valid = ['REGISTER', 'NOTIFY']
+
+    def instance(self, packet):
+        pass
+
+    def register(self, packet):
+        logger.info('Register')
+        print 'Registration Packet:'
```
Comment (kfdm) [inline regrowl/bridge/netcent.py]: I imagine most of these print statements should probably be switched to regular logger. statements? I think only the test "echo" bridge should probably use print.

**Comment by kfdm** [comment]
Great idea! I'm not sure why I didn't think about bridging to Notification Center.

**Comment by heilerich** [comment]
Done. I hope everything is well-ordered enough for you ;)

**Comment by kfdm** [comment]
Thanks :)

**Pull request merged**